global.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Are we completely off .net Core 2/3 now? Or are we still using it anywhere?

If we are fully onto .net 5, can you update this Issue:
Update test code to .net 6 #6487

We should still update to .net 6 at some point, but now that we are on .net 5, it is much less urgent.